### ISSUE TITLE
Picker Component does not announce "disabled"

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30940]
## Description
When using a screen reader the picker component does not announce that it is disabled.

This issue is covered in-depth by parent issue #30840, please check there for more details.

## React Native version:
0.63

**Comment by Simek** [comment]
@amarlette Picker component has been deprecated in the `0.62` release (https://github.com/facebook/react-native/commit/9349313c995a02b0a9de1e1692288dcd200a84b8) and have been extracted from the core. 

Please report the issue in this repository:
* https://github.com/react-native-picker/picker

**Comment by amarlette** [comment]
Closing this issue because the component has been deprecated. 

Issue opened with respective repository https://github.com/react-native-picker/picker/issues/215
